rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -279,20 +289,10 @@ s32 cellGameDataSetSystemVer(vm::cptr<char> systemVersion)
 	return CELL_OK;
 }
 
-s32 cellGameDataExitBroken()
+error_code cellGameDataExitBroken()
 {
 	cellGame.warning("cellGameDataExitBroken()");
-
-	s32 res = open_msg_dialog(CELL_MSGDIALOG_TYPE_SE_TYPE_ERROR | CELL_MSGDIALOG_TYPE_BUTTON_TYPE_OK | CELL_MSGDIALOG_TYPE_DISABLE_CANCEL_ON,
-		vm::make_str("There has been an error!\n\nPlease delete the game's game data."));
-
-	if (res != CELL_OK)
-	{
-		return CELL_GAMEDATA_ERROR_INTERNAL;
-	}
-
-	sysutil_send_system_cmd(CELL_SYSUTIL_REQUEST_EXITGAME, 0);
-	return CELL_OK;
+	return open_exit_dialog("There has been an error!\n\nPlease delete the game's game data.", true);
```

Comment:
To make it more clear i'd reword it to:
"There has been an error!\n\nPlease remove game data for this game."

Reply:
If im not mistaken those messages come from the ps3, we didnt written it ourselfs.

Reply:
Does not mean we can't improve on them, don't we?

Reply:
I don't think these are official message texts.
The wording in cellGameContentErrorDialog is actually kinda weird.
I think the hints in braces are just a note to developers.
So I renamed them to make more sense.

I will change the others too unless someone knows the correct wording